config/ingestCalibs.py

```diff
@@ -1,6 +1,8 @@
 from lsst.obs.decam.ingestCalibs import DecamCalibsParseTask
 config.parse.retarget(DecamCalibsParseTask)
 config.parse.hdu = 1
+config.parse.translation = {"obstype": "OBSTYPE"}
```

Comment:
Where is this used? Isn't `OBSTYPE` obtained directly in `getCalibType`? If I understand correctly, this line should not be needed.